pkg/virt-launcher/virtwrap/network/network.go

```diff
@@ -131,7 +131,7 @@ func SetupPodNetworkPhase2(vmi *v1.VirtualMachineInstance, domain *api.Domain) e
 
```

Comment:
I don't understand the title and message of this commit; and neither it's motivation

Reply:
What's the name of it ?

Reply:
Added description

Reply:
Yeah, the commit msg is weird. Promote ? 

I would instead specify that the network handler is now a member of the podNic struct.

Reply:
Done